chapters/VK_KHR_shared_presentable_image/wsi.txt

```diff
@@ -55,8 +55,7 @@ include::{generated}/api/protos/vkGetSwapchainStatusKHR.txt[]
 include::{generated}/validity/protos/vkGetSwapchainStatusKHR.txt[]
 --
 
-The possible return values for fname:vkGetSwapchainStatusKHR should: be
-interpreted as follows:
+The meanings of status codes returned by flink:vkGetSwapchainStatusKHR are:
```

Comment:
nit: unnecessary nounification

I would prefer:

> A status code returned by `vkGetSwapchainStatusKHR` means the following:

Reply:
agreed. that's better

Reply:
reworded

Reply:
Seems this is satisfactory.